data/categories.txt

```diff
@@ -6093,67 +6093,651 @@ sw:Bwawa la kuogelea
 fa:استخرشنا
 mr:जलतरण तलाव
 
-sport-equestrian
-en:Equestrian Sports
-ru:Конный спорт
-bg:Коне|спорт|езда
-nl:Paardensport|ruitersport
-ko:승마|마술 스포츠
-pl:Jazda konna
-fr:Sport équestre
-hu:Lovassportok
-de:Pferdesport
-it:Equitazione
-es:Deportes ecuestres
-eu:Zaldi-kirolak
-pt:Desportos equestres
-pt-BR:Esportes equestres
-tr:Binicilik
-el:Ιππασία
-mr:घोडेस्वार खेळ
+sport-american_football
+en:American Football
+ar:كرة القدم الأمريكية
+be:Амерыканскі футбол
+cs:Americký fotbal
+da:Amerikansk fodbold
+de:Amerikanischer Fußball
+el:αμερικάνικο ποδόσφαιρο
+es:Fútbol americano
+es-MX:Fútbol americano
+eu:Futbol amerikarra
+fa:فوتبال آمریکایی
+fi:Amerikkalainen jalkapallo
+fr:Football américain
+he:כדורגל אמריקאי
+hu:Amerikai foci
+id:Sepak Bola Amerika
+it:Football americano
+ja:アメリカンフットボール
+ko:미식 축구
+mr:अमेरिकन फुटबॉल
+nb:Amerikansk fotball
+nl:Amerikaans voetbal
+pl:Futbol amerykański
+pt:Futebol americano
+pt-BR:Futebol americano
+ro:Fotbal american
+ru:Aмериканский футбол
+sk:Americký futbal
+sv:Amerikansk fotboll
+sw:Soka ya Marekani
+th:อเมริกันฟุตบอล
+tr:Amerikan futbolu
+uk:Американський футбол
+vi:Bóng bầu dục Mỹ
+zh-Hans:美式足球
+zh-Hant:美式足球
+
+sport-archery
+en:Archery
+ar:الرماية
+be:Стральба з лука
+cs:Lukostřelba
+da:Bueskydning
+de:Bogenschießen
+el:Τοξοβολία
+es:Tiro al arco
+es-MX:Tiro al arco
```

Comment:
If the translation is the same as the base lang (es or pt), then it should not be overridden. @pastk right?

Reply:
Its not necessary anymore, but it won't hurt either.
Our translation files are full of duplicates like this from the [recent] times when duplicating was mandatory.
It should be deduplicated by an autoformatting script some day.

Reply:
There is no reason to create new duplicates and support them. Fewer lines = easier support.

Reply:
True, my point is just its tedious and inefficient to do by hand ("мартышкин труд"), so I would rather accept as is instead of insisting people spend their time on insignificant things which could be fixed later in bulk.

Reply:
Consistency even with minor details makes a good example for other contributors.

Reply:
@biodranik @pastk I can try to create a simple script to de-duplicate non-base strings if base string value is the same. Are you fine with it?

Reply:
@dbf256 That'd be great!
So the de-duplication shall apply to regional languages only, e.g. `es-MX`, `en-AU`, `pt-BR` etc. See #1757 for a reference. TBH I don't remember if `zh-Hant`/`zh-Hans` should follow this rule too or not - need to check...

(you may want to re-use that outdated `categories_converter.py` script if its of any value)
And then we'll be able to expand the script to add auto-formatting, validation, etc. to categories.txt too!

Reply:
Also - if you want and you have time - it'll be great to do the same de-duplication to `types_strings.txt` and `strings.txt`, its better to add this functionality to an existing script `strings_utils.py` which processes those files.

Reply:
Here is the full picture:
1. zh-Hans and zh-Hant are distinct scripts, they always should be present.
2. All string validations and cleanups should be automatically run from tools/unix/generate_localizations.sh
3. Existing duplicates should be cleaned when sub-lang (en-AU for en, es-MX for es, pt-BR for pt) has the same translation as the base lang.
4. Ensure that categories.txt takes all translations from types_strings.txt, and has only additional synonyms where necessary.
5. After that categories.txt should be cleaned automatically, leaving only those synonyms that are absent in types_strings.txt